src/main/java/ejercicio/banco/repository/MySqlAccountRepository.java

```diff
@@ -1,16 +1,14 @@
 package ejercicio.banco.repository;
 
 import ejercicio.banco.dto.Account;
+import ejercicio.banco.util.DataBaseUtil;
 
 import java.sql.Connection;
 import java.sql.PreparedStatement;
 import java.sql.ResultSet;
 import java.util.ArrayList;
 import java.util.List;
 
-import static ejercicio.banco.util.DataBaseUtil.closeConections;
-import static ejercicio.banco.util.DataBaseUtil.startConnection;
```

Comment:
Sorry, I did not see these imports. I thought you were using a private method. It was ok then :)